arm/subscriptions/version.go

```diff
@@ -25,11 +25,11 @@ import (
 const (
 	major = "0"
 	minor = "1"
-	patch = "0"
+	patch = "1"
```

Comment:
if you haven't tagged you probably shouldn't bump the version up. just undo the change in `patch`.